man/kernel-command-line.xml

```diff
@@ -151,6 +157,8 @@
           <citerefentry><refentrytitle>systemd-volatile-root.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
           and
           <citerefentry><refentrytitle>systemd-fstab-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry>.</para>
+
+          <xi:include href="version-info.xml" xpointer="v233"/>
```

Comment:
Some values (i.e. `tmpfs` and `overlay`) were added later. How do we want to reflect that? In this case, some more edits are needed, since the second sentence of this section says

> Takes a boolean argument, or the special value `state`.

which is not true anymore, since it also accepts `tmpfs` and `overlay`.